internal/criocli/criocli.go

```diff
@@ -1012,6 +1015,11 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 			Usage:   "The number of seconds between collecting pod and container stats. If set to 0, the stats are collected on-demand instead.",
 			EnvVars: []string{"CONTAINER_STATS_COLLECTION_PERIOD"},
 		},
+		&cli.BoolFlag{
+			Name:    "disable-criu-support",
+			Usage:   "Disable CRIU integration even if the criu binary is found in $PATH.",
+			EnvVars: []string{"CONTAINER_DISABLE_CRIU_SUPPORT"},
+		},
```

Comment:
How about inverting the logic by having `enable-criu-support` default to `true`?

Reply:
I am not sure I get it.

The idea was to automatically enable CRIU support if the binary is found and have the possibility to disable it.

Maybe you mean something like this: `--enable-criu-support false` to disable it? But then it wouldn't be a boolean parameter. I think. Not sure. If you could provide some more details what you mean that would help me.

Reply:
Ah, okay. Thanks. But (:wink:), the help text now sounds a bit strange. It is enabled by default and to disable it I need to specify `--enable-criu-support=false`, right? This seems odd. I am happy to do the change, it just feels strange.

Reply:
> It is enabled by default and to disable it I need to specify `--enable-criu-support=false`, right? 

Yeah.

> This seems odd. I am happy to do the change, it just feels strange.

Personally, it's not odd to me. But since you mentioned that it feels strange we probably have to consider the `disable-` flag. Let's pull in @haircommander, WDYT?

Reply:
Sounds reasonable.  I will update the PR accordingly. Thanks for the clarification.

Reply:
Changed it to --enable-criu-support with the default set to disabled.